test/colony-network-mining.js

```diff
@@ -4290,11 +4165,8 @@ contract("ColonyNetworkMining", accounts => {
       const nInactiveLogEntries = await repCycle.getReputationUpdateLogLength();
       assert.equal(nInactiveLogEntries.toNumber(), 13);
 
-      badClient = new MaliciousReputationMinerExtraRep(
-        { loader: contractLoader, minerAddress: OTHER_ACCOUNT, realProviderPort: REAL_PROVIDER_PORT, useJsTree },
-        40, // Skill 4
-        "0xfffffffff"
-      );
+      // Skill 4
```

Comment:
I added the comments alongside some of the bad clients `entityToFalsify` to try and deal with some of the uncertainty around verifying which entry we are falsifying so at least when we refactor we can use that bit of information without having to log the good client updates first to understand the intention. Unless we find a better way to verify these (as described [here](https://github.com/JoinColony/colonyNetwork/issues/317#issuecomment-416025328) ) I suggest we add this information to all bad clients instantiations. It probably shouldn't be done as part of this PR but still, worth mentioning it here.

Reply:
👍